org/minipad/index.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: org
+title: minipad
```

Comment:
This should be `minisbett`

Reply:
but isn't it supposed to be the organization name?

Reply:
You mean the github organization name? Then it's "minipadkb".

Reply:
It doesn't need to match your github organization. The pid's `owner` needs to match the org folder name.